packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -943,10 +991,16 @@ export class GarbageCollector implements IGarbageCollector {
     private logIfInactive(
         eventSuffix: "Changed" | "Loaded" | "Revived",
         nodeId: string,
-        timestampMs: number,
+        currentReferenceTimestampMs = this.getCurrentReferenceTimestampMs(),
         packagePath?: readonly string[],
         requestHeaders?: IRequestHeader,
     ) {
+        // If there is no reference timestamp to work with, no ops have been processed after creation. If so, skip
```

Comment:
Is this situation in itself interesting to track?

Reply:
I think it's interesting but not from GC's point of view. If we are summarizing in the absence of ops, we should log an event during summary saying this is happening and its unexpected.
I am currently adding some telemetry related to the summary process and I can add this too.

Reply:
Added telemetry if we summarize in absence of ops here - https://github.com/microsoft/FluidFramework/pull/9225